internal/config/config.go

```diff
@@ -46,12 +46,14 @@ func init() {
 	vip.SetDefault(DbInsecure, true)
 	vip.SetDefault(AwsRegion, "eu-central-1")
 	vip.SetDefault(TorProxyUrl, "127.0.0.1:9050")
+	vip.SetDefault(RegistryUrl, "https://api.github.com/repos/tdex-network/tdex-registry/contents/registry.json")
```

Comment:
is this better?

`https://raw.githubusercontent.com/tdex-network/tdex-registry/master/registry.json`

Reply:
Yeap, Done